native_client/BUILD

```diff
@@ -52,15 +67,22 @@ tf_cc_shared_object(
             "kiss_fft130/_kiss_fft_guts.h",
             "kiss_fft130/tools/kiss_fftr.h",
             "ds_version.h"] +
-           DECODER_SOURCES,
+            select({
```

Comment:
nit: Instead of doing this select here, you can do it at the definition of `DECODER_SOURCES` and then use a single variable in the rest of the file.